src/aaudio/AAudioLoader.cpp

```diff
@@ -229,4 +229,110 @@ AAudioLoader::signature_I_PSKPLPL AAudioLoader::load_I_PSKPLPL(const char *funct
     return reinterpret_cast<signature_I_PSKPLPL>(proc);
 }
 
+// Ensure that all AAudio primitive data types are int32_t
+#define ASSERT_INT32(type) static_assert(std::is_same<int32_t, type>::value, \
+#type" must be int32_t")
+
+#define SAMSG "Oboe constants must match AAudio constants."
```

Comment:
what is `SAMSG`?

Reply:
Ah, it's an abbreviation of STATIC_ASSERT_MESSAGE. Please change the name to make this clearer.

Reminds me of this: http://reviewthecode.blogspot.com/2016/01/wtf-per-minute-actual-measurement-for.html

Reply:
Maybe ERRMSG